recipes/biogridpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: biogridpy
+  version: "0.1.1"
+
+source:
+  fn: biogridpy-0.1.1.tar.gz
+  url: https://pypi.python.org/packages/97/3f/389d2928ed5d59a4fde16a506e8c6e0e04620f7f61fc84caefca2aa7d84d/biogridpy-0.1.1.tar.gz
+  md5: 5a39971ec8517658de774dd0b0919a4e
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - biogridpy = biogridpy:main
+    #
+    # Would create an entry point called biogridpy that calls biogridpy.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - biogridpy
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
```

Comment:
I initially used the chango recipe as a template.  This commit removed the unnecessary comments, was that what you were referring to?  Thanks for reviewing!